mews-operations/availabilityBlockProcessor.md

```diff
@@ -0,0 +1,176 @@
+| `spaceCategories` | [`Space category allocation`](#space-category-allocation) collection | required | Allocated categories of the availability block. |
+| `booker` | [`Customer`](./reservations.md#customer) object | required | The main booker. This does not mean that the person has arrived at the property. |
+| `company` | [`Company`](./reservations.md#company) object | optional | The company associated with the availability block. |
+| `notes` | `string` | optional | It's a notes. |
+
+
+#### Dates
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `yyyy-mm-dd`, e.g. `2024-12-05`, this is the first day of the block and so can be booked for stay. |
+| `end` | `string` | required | End date in format `yyyy-mm-dd`, e.g. `2024-12-08`; this day is _not_ included in the block and so cannot be booked for stay (it is analogous to reservation departure). |
+
+
+#### Release strategy
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `type` | `string` | [`Release strategy type`](#release-strategy-type) required | release strategy for the availability block. |
+| `fixedDate` | `string` | required | release date in format `"yyyy-MM-dd"` \(e.g., `"2024-12-08"`\). |
+
+
+#### Space category allocation
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `spaceTypeCode` | `string` | required | Space type code of the block. |
+| `allocatedSpaces` | [`Space count`](#space-count) collection | required | The number of spaces **allocated** in the time interval. |
+| `rates` | [`Rates`](#rate) collection | optional | Prices for the space category. |
+
+
+### Space count
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2024-12-05"`\). |
+| `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2024-12-08"`\). |
+| `count` | `int` | required | Number of spaces (base of usage either occupied or allocated.) |
+
+
+### Rate
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2024-12-05"`\). |
+| `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2024-12-31"`\). |
+| `grossAmount` | `string` | required | Price with taxes included. | 
+
+
+#### Status
+
+* `Created`
+* `Updated`
+* `Canceled`
+
+#### Release strategy type
+
+* `Fixed`
+
+### Response
+
+[Synchronous simple response](../guidelines/responses.md#synchronous-simple-response) is expected to determine whether the operation was accepted or not.
+
+[[Async] availability block processing confirmation](#availability-block-processing-confirmation) is expected to determine whether the operation was processed afterward or not.
+
+### Availability block processing confirmation
+```json
+{
```

Comment:
The indentation here is different to the standard, e.g. see above on the same page.